contributors/guide/first-contribution.md

```diff
@@ -19,69 +19,113 @@ description: |
-then the robot will assign the issue to you and your name will present at `Assignees` list.
+When you've found an issue to work on, you can assign it to yourself.
 
-### Learn about SIGs
+* Reply with `/assign` or `/assign @yourself` on the issue you'd like to work on 
+* The [K8s-ci-robot] will automatically assign the issue to you. 
+* Your your name will then be listed under, `Assignees`.
 
-You may have noticed that some repositories in the Kubernetes Organization are owned by [Special Interest Groups](/sig-list.md), or SIGs.
-We organize the community into SIGs in order to improve our workflow and more easily manage what is a very large community project.
-The developers within each SIG have autonomy and ownership over that SIG's part of Kubernetes. Check out the [list of SIGs](/sig-list.md) for contact information.
+## Learn about SIGs
 
-Understanding how to interact with SIGs is an important part of contributing. 
+Some repositories in the Kubernetes Organization are owned by 
+[Special Interest Groups], or SIGs.
 
-#### SIG structure
+The Kubernetes community is broken out into SIGs in order to improve its workflow,
+and more easily manage what is a very large community project.
+
+The developers within each SIG have autonomy and ownership over that SIG's part 
+of Kubernetes. 
+
+Check out the [list of SIGs] for contact information.
+
+Understanding how to interact with SIGs is an important part of contributing 
+to Kubernetes. 
+
+### SIG structure
 
 A SIG is an open, community effort.
-Anybody is welcome to jump into a SIG and begin fixing issues, critiquing design proposals and reviewing code.
-SIGs have regular [video meetings](https://kubernetes.io/community/) which everyone is welcome to.
-Each SIG has a slack channel, meeting notes, and their own documentation that is useful to read and understand.
 
-There is an entire SIG ([sig-contributor-experience](/sig-contributor-experience/README.md)) devoted to improving your experience as a contributor.
-Contributing to Kubernetes should be easy.
-If you find a rough edge, let us know! Better yet, help us fix it by joining the SIG; just
-show up to one of the [bi-weekly meetings](https://docs.google.com/document/d/1qf-02B7EOrItQgwXFxgqZ5qjW0mtfu5qkYIF1Hl4ZLI/edit).
+Anybody is welcome to jump into a SIG and begin fixing issues, critiqe design 
+proposals, and review code.
+SIGs have regular [video meetings] which everyone is welcome to attend.
+
+Each SIG has a Slack channel, meeting notes, and their own documentation that 
+is useful to read and understand.
+
+There is an entire SIG ([sig-contributor-experience]) devoted to improving your 
+experience as a contributor.
 
-#### Find a SIG that is related to your contribution
+If you have an idea for improving the contributor experience, please consider
+attending one of the Contributor Experience SIG's [bi-weekly meetings].
```

Comment:
I can't offer a suggestion on this block through the GitHub UI, but it can be consolidated down.
the `bi-weekly meetings` should also be updated to `weekly`. I missed that when originally looking it over^^;;;